ports/openjpeg/vcpkg.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "openjpeg",
+  "version-string": "2.3.1",
+  "port-version": 4,
```

```suggestion
  "version-semver": "2.3.1",
  "port-version": 3,
```
For package versions, you can refer to this doc https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions.

For port-version, since it was 2 in last change, it should be 3 here.